app/controllers/miq_policy_set_controller.rb

```diff
@@ -83,7 +83,7 @@ def profile_cancel
   end
 
   def profile_save_add
-    assert_privileges("profile_#{params[:id] ? "edit" : "new"}")
+    assert_privileges("miq_policy_set_#{params[:id] ? "edit" : "new"}")
```

Comment:
Now it's fixed for real.. thanks @Fryguy 

![image](https://user-images.githubusercontent.com/19339/119015888-57c17b80-b967-11eb-8dbe-d1df4ecc6add.png)

![image](https://user-images.githubusercontent.com/19339/119015953-6445d400-b967-11eb-82d0-9a8c2ff9a40e.png)